operations/research-and-development/organization/release_devops.md

```diff
@@ -1,23 +1,51 @@
-# Release DevOps
-
-The Release DevOps team is responsible for the release artifacts, pipelines, and processes for the release of Mattermost products. 
-
-## Areas of Ownership
-
-- Release pipelines, artifacts and processes (release branches/tags) for:
-  - Self-Managed
-  - Mobile
-  - Plugins & Apps
-  - CLI (mmctl)
-- Release for Distribution channels
-  - GitLab Omnibus
-  - Mattermost Omnibus
-- Release principles and processes
-  - Extended Support Releases
-- Incident response/hotfixes
-- Release automation for release processes (mattermod, matterbuild)
-- Maintain/scale day to day development pipelines/artifacts/images
-- Reliability/Observability of Push Proxy
-- Reliability/Observability of company’s GitLab
-- Maintain Weblate
-- GitHub Management
+# Release (DevOps)
+
+## Mision
+
+The Release (DevOps) team enables Mattermost Engineering to deliver features in a safe, scalable and efficient fashion to Cloud and self-managed customers. 
+The team ensures that Mattermost weekly, monthly, security, and patch releases are publicly released in a timely fashion.
+
+## Vision
+
+By its own nature, the Release (DevOps) team is a backstage, non-user feature facing team whose product and output has a high impact on day to day development operations and Mattermost releases. The team creates the workflows, frameworks, tools (ChatOps, Bots), architecture and automation for Engineering teams to see their work reach production effectively and efficiently. The team is focused on our CI/CD blueprint by driving the necessary changes in our software development processes and workflows, as well as infrastructure changes, to embrace the benefits of CI/CD.
```

Comment:
Curious: What does "non-user feature facing team" mean? I'm unfamiliar with this phrase.

Reply:
we haven't any UI features (actually borrowed from GitLab) but CLIs, pipelines etc.